src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md

```diff
@@ -14,6 +14,10 @@
   occurs.
   ([#3407](https://github.com/open-telemetry/opentelemetry-dotnet/issues/3407))
 
+* Create spans for HTTP retries and redirects according to
+  [opentelemetry-specification #2078](https://github.com/open-telemetry/opentelemetry-specification/pull/2078).
+  ([#3072](https://github.com/open-telemetry/opentelemetry-dotnet/pull/3072))
```

Comment:
also be specific that this is only done to the HttpClient .NET Core one, and .NET Framework.